pillar/nginx/starcellbio.sls

```diff
@@ -0,0 +1,74 @@
+{% set app_name = 'starcellbio' %}
+{% set env_settings = salt.cp.get_file_str("salt://environment_settings.yml")|load_yaml %}
+{% set ENVIRONMENT = salt.grains.get('environment', 'rc-apps') %}
+{% set env_data = env_settings.environments[ENVIRONMENT] %}
+{% set server_domain_names = env_data.purposes['odl-video-service'].domains %}
+{% set ovs_login_path = 'login' %}
+
+nginx:
+  ng:
+    install_from_source: True
+    source_version: 1.13.8
+    source_hash: 8410b6c31ff59a763abf7e5a5316e7629f5a5033c95a3a0ebde727f9ec8464c5
+    certificates:
+      odl_wildcard:
+        public_cert: __vault__::secret-operations/global/odl_wildcard_cert>data>value
+        private_key: __vault__::secret-operations/global/odl_wildcard_cert>data>key
+    servers:
+      managed:
+        {{ app_name }}:
+          enabled: True
+          config:
+            - server:
+                - server_name: {{ server_domain_names }}
+                - listen:
+                    - 80
+                - listen:
+                    - '[::]:80'
+                - location /:
+                    - return: 301 https://$host$request_uri
+            - server:
+                - server_name: {{ server_domain_names }}
+                - listen:
+                    - 443
+                    - ssl
+                    - default
+                - listen:
+                    - '[::]:443'
+                    - ssl
+                - root: /opt/{{ app_name }}/
+                - ssl_certificate: /etc/nginx/ssl/odl_wildcard.crt
+                - ssl_certificate_key: /etc/nginx/ssl/odl_wildcard.key
+                - ssl_stapling: 'on'
+                - ssl_stapling_verify: 'on'
+                - ssl_session_timeout: 1d
+                - ssl_session_tickets: 'off'
+                - ssl_protocols:
+                    - TLSv1
+                    - TLSv1.1
```

Comment:
Should we remove both of these?

Reply:
I suppose we can give it a try, but I anticipate that leading to a support request...